examples/distance-matrix/README.md

```diff
@@ -0,0 +1,35 @@
+# Basic Google Maps Setup Example
```

Comment:
Shouldn't the title refer to the example?

```suggestion
# `useDistanceMatrix` Hook Setup Example
```

(Please check other READMEs as well. I think they have all the same title.)

Reply:
Oopsi - I'll change the rest in a new PR